classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    student.classroom = self
+    @students.push(student)
+  end
```

Comment:
- [ ] As requested by the previous reviewer to implement `Both ways relationships`. The below-suggested code should be placed in the Student class to achieve this.

```ruby
def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end